BrowserKit/Sources/ToolbarKit/ToolbarManager.swift

```diff
@@ -25,15 +26,17 @@ public class DefaultToolbarManager: ToolbarManager {
 
     public func getAddressBorderPosition(for toolbarPosition: AddressToolbarPosition,
                                          isPrivate: Bool,
+                                         isWebPage: Bool,
                                          scrollY: CGFloat) -> AddressToolbarBorderPosition {
         // display the top border if
         // - the toolbar is displayed at the bottom
         // display the bottom border if
-        // - the toolbar is displayed at the top and the website was scrolled
+        // - the toolbar is displayed at the top and page is scrolled (applies to homepage)
+        // - the toolbar is displayed at the top and we are on a webpage
```

Comment:
I am confused about this change. Is this coming from design? In [Figma](https://www.figma.com/design/8fsCvtnweBkWGZ9NfL907l/__-Toolbar-Redesign?node-id=7770-248638&t=NmC9MBZhS50EXAza-4) the requirements for websites are the following:
<img width="535" alt="Screenshot 2024-09-25 at 10 23 52" src="https://github.com/user-attachments/assets/8af96c38-2274-497e-8545-d51bef42235c">
So the bottom border for websites should only be displayed on scroll unless the user is in private mode - only then it's displayed always.